src/components/NodeToolbar/NodeToolbar.js

```diff
@@ -257,30 +254,6 @@ class NodeToolbar extends React.Component<Props, State> {
     );
   }
 
-  renderInlineWheelchairAccessibilityEditor() {
```

Comment:
@saryekad This has to be merged with @mutaphysis' new code here: https://github.com/sozialhelden/wheelmap-frontend/pull/246/files#diff-5657e5c46dbcbe59ddafaa650f1f828eR262